stretch_core/stretch_core/trajectory_components.py

```diff
@@ -0,0 +1,62 @@
+from hello_helpers.hello_misc import to_sec
+
+
+class TrajectoryComponent:
+    def __init__(self, name, trajectory_manager):
+        self.name = name
+        self.trajectory_manager = trajectory_manager
+
+    def get_position(self):
+        return self.trajectory_manager.status['pos']
+
+    def get_velocity(self):
```

Comment:
Why don't we have a getter for acceleration

Reply:
I haven't needed it yet.

Reply:
Also, more importantly, acceleration isn't reported by the drivers.

Reply:
That's fair.